app/jobs/assignment_repo/porter_status_job.rb

```diff
@@ -38,13 +38,15 @@ def perform(assignment_repo, user)
             status: assignment_repo.assignment.invitation&.status
           )
           logger.warn result.to_s
+          GitHubClassroom.statsd.increment("v2_exercise_repo.import.fail")
         when Creator::REPOSITORY_CREATION_COMPLETE
           assignment_repo.assignment.invitation&.completed!
           ActionCable.server.broadcast(
             RepositoryCreationStatusChannel.channel(user_id: user.id),
             text: result,
             status: assignment_repo.assignment.invitation&.status
           )
+          GitHubClassroom.statsd.increment("v2_exercise_repo.import.success")
         end
       rescue Octopoller::TimeoutError
         PorterStatusJob.perform_later(assignment_repo, user)
```

Comment:
Do you think it might be useful to report here as well?

Reply:
Yes! This also raises the question, should we be reporting retries on `CreateGitHubRepositoryJob`? I feel we should because technically the success rate of the new import resiliency pattern is `net success = success - (failure  - retries)`